### DISCUSSION TITLE
Read MAC address from client PC

### DISCUSSION THREAD

**Discussion by chrisrusso** [tauri-apps/tauri#7864] [General]
Hi everyone!

Is it possible to read the MAC address from the client computer where the Tauri software is installed?

Thanks in advance!
Chris

**Comment by FabianLars** [comment]
This rust crate looks pretty solid to me https://crates.io/crates/mac_address - you could use it inside a [command](https://tauri.app/v1/guides/features/command/)

**Reply by chrisrusso** [reply]
Thanks Fabian!!! We found that the best way to communicate with the lower level was using commands, indeed... appreciated!
